src/librustc/ty/sty.rs

```diff
@@ -646,6 +646,17 @@ impl<'tcx> PolyExistentialTraitRef<'tcx> {
 pub struct Binder<T>(pub T);
 
 impl<T> Binder<T> {
+    /// Wraps `value` in a binder, asserting that `value` does not
+    /// contain any bound regions that would be bound by the
+    /// binder. This is commonly used to 'inject' a value T into a
+    /// different binding level.
+    pub fn new_not_binding<'tcx>(value: T) -> Binder<T>
```

Comment:
(Personally i'd call this `dummy_binding` or `new_dummy_binding` (since in my head the binding exists, it just is useless), but I can cope with this function name.)

Reply:
I could get behind `Behinder::dummy` or something.